tests/test_view_helpers.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from django.http import Http404
+import pytest
+
+from nomination import views, models
+import factories
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestGetProject():
+
+    def test_returns_project(self):
+        project = factories.ProjectFactory()
+        result = views.get_project(project.project_slug)
+        assert result == project
+
+    def test_raises_404(self):
+        with pytest.raises(Http404):
+            views.get_project('fake_project')
+
+
+class TestGetProjectUrlCount():
+
+    def test_returns_surt_count(self):
+        factories.SURTFactory.create_batch(10)
+        # Should not affect the surt count.
```

Comment:
No, I'm saying that it shouldn't. Basically wherever these comments are, I have created extra objects that should not appear in the results. I do that just to make sure that the we are not getting _more_ than we expect to get in the results.